Corona-Warn-App/src/test/java/de/rki/coronawarnapp/covidcertificate/vaccination/core/VaccinationTestData.kt

```diff
@@ -174,11 +191,18 @@ class VaccinationTestData @Inject constructor(
         expiresAt = Instant.parse("2022-05-11T09:23:03.000Z"),
     )
 
-    val personBVac1CertificateData = DccData(
+    private val personBVac1CertificateData = DccData(
```

Comment:
Why make these private? Can't we no longer access them in tests then?

Reply:
IDE: could be private ?
Me: Yes, why not 😀